engine/apps/mobile_app/alert_rendering.py

```diff
@@ -10,9 +10,17 @@ def _render_for(self):
 
 
 def get_push_notification_subtitle(alert_group):
+    MAX_ALERT_TITLE_LENGTH = 200
+    print(f"ALERTS COUNT: {alert_group.alerts.count}")  # TODO: debugging test. remove after debugging
     alert = alert_group.alerts.first()
+    print(f"ALERT TITLE: {alert.title}, DATA: {alert.raw_request_data}")  # TODO: debugging test. remove after debugging
     templated_alert = AlertMobileAppTemplater(alert).render()
+    print(f"TEMPLATED ALERT: {templated_alert}")  # TODO: debugging test. remove after debugging
```

Comment:
remove `print` statements

Reply:
👍  I'm debugging failing tests, that work locally